receiver/prometheusreceiver/internal/transaction_test.go

```diff
@@ -365,6 +572,11 @@ func TestMetricBuilderGauges(t *testing.T) {
 				pt0.SetStartTimestamp(0)
 				pt0.SetTimestamp(tsNanos)
 				pt0.Attributes().PutString("foo", "bar")
+				e0 := pt0.Exemplars().AppendEmpty()
+				e0.SetTimestamp(pcommon.NewTimestampFromTime(time.UnixMilli(1663113420863)))
```

Comment:
**NEVER** unittest with the same code. Use pre-calculated input and output values. If you use an already tested code like timestampFromMs then is ok, since we know that that code is correct.

Reply:
Done